apps/site/pages/docs/api-extensions/consuming-api-extensions.mdx

```diff
@@ -0,0 +1,137 @@
+---
+title: 'Consuming FastStore API extension with custom components'
+sidebar_label: 'API Extensions'
+---
+
+import { Callout, Step, Steps } from 'nextra-theme-docs'
+
+<header>
+
+# Consuming FastStore API extension with custom components
+
+</header>
+
+In this guide, learn how to consume new fields from [custom sections](/docs/building-sections/creating-a-new-section) and [overridable components](/docs/building-sections/overriding-components-and-props).
+
+FastStore exposes the data that comes from FastStore API along with FastStore API Extensions inside a provider called `PageProvider`.
+This data comes as a `context` in `PageProvider`, and you can use the following hooks to access it:
+
+
+<ul>
```

Comment:
The use cases are clear 🚀 

I'd change the examples so every hook is called from within a component, as it should always be according to React rules.

Reply:
We can see how it looks, but I think it might look better just repeating the code for each example. It'd be more straightforward for readers I guess.